source/MRViewer/MRUIStyle.h

```diff
@@ -45,6 +45,38 @@ struct ButtonIconCustomizationParams : public ButtonCustomizationParams
     bool flatBackgroundColor = false;
 };
 
+struct PlotAxis
+{
+    enum class TypeAxis
+    {
+        horizontal,
+        vertical,
+        none
+    } axisType = TypeAxis::none;
+    // the point from which the axes will be drawn
+    ImVec2 startAxisPoint;
+    // length dash without text
+    float lenDash = 8.0f;
+    // length dash with text
+    float lenDashWithText = 12.0f;
+    // text offset from dash
+    float textPadding = 3.0f;
+
+    // offset from the starting point along the axis
+    float offset = 0;
+    // the number of dash on the axis
+    size_t numDash = 5;
+    // the distance between the dash on the axis
+    float step = 0;
```

Comment:
isn't it plotSize/numDash ?

Reply:
yes, but there is no size parameter in the structure, for more flexible customization